primitives/src/lib.rs

```diff
@@ -472,7 +472,7 @@ create_currency_id! {
     #[repr(u8)]
     pub enum TokenSymbol {
         DOT("Polkadot", 10) = 0,
-        INTERBTC("interBTC", 8) = 1,
+        IBTC("interBTC", 8) = 1,
```

Comment:
shouldn't this be ibtc as well?

Reply:
No we're keeping "interBTC" as is, just changing the ticker.